ratlib/languages.py

```diff
@@ -0,0 +1,533 @@
+class Language:
+    """Static class to hold some functions for handling languages"""
+    @staticmethod
+    def name(code: str, raise_error: bool=True, return_all: bool=False) -> str or tuple or None:
+        """
+        Get language name corresponding to provided ISO 639-1 code
+        :param code: ISO 639-1 code
+        :param raise_error: Raise error if language not found (rather than return None)
+        :param return_all: If False, and the language has multiple names, only return the first
+        :return: Language name
+        """
+        try:
+            result = languages[code]
+            if isinstance(result, str):
+                return result
+            else:
+                return result if return_all else result[0]
+
+        except KeyError as e:
```

Comment:
please avoid introducing single-char variables. please rename this to `ex` or something a little more descriptive.
see [here](https://github.com/FuelRats/tech-rat-code-standards) for more information

Reply:
Again, usually I would agree with you. However using `e` to represent an exception being handled - like using `i` as an index while iterating over something or `x, y` for coordinates - is quite common practice.

Reply:
the point of that rule is to keep variables easier to read and clear of their purpose. `err` or `error` would be the best names for an error.

Reply:
just because its `common practice` does not make it any more readable. I shouldn't have to look back to where `e` or `i` was declared to determine what the heck its for.

Reply:
Since you're the senior developers on this project I'll bow to your opinion (Though I still disagree - In small scopes, I find clear single-character variable names acceptable.)

Reply:
I'd like to make clear here that we do have a standard, and contributors are expected to abide by them in the same way that any other OSS org expects. To put it extremely blunt, It's fine that you have an opinion, __but it does not matter, and your opinions will never override the standard set by the org.__

Reply:
I never suggested that they did.